### ISSUE TITLE
Bug with ReLU followed by adding Tensors (converting from pytorch)

### ISSUE THREAD

**Issue by HeekonKim** [NVIDIA/TensorRT#652]
## Description

Hi, I am using TensorRT converting from pytorch and I found that my model behaves quite different with PyTorch/ONNX when converted to TensorRT.

After investigating the issue, I found that ReLU operation does not apply appropriately at specific situations.

(In my case within residual block) **ReLU operation is ignored when followed by adding Tensors.**

## Environment

**TensorRT Version**: 7.0.1.1
**GPU Type**: GTX 1080 Ti
**Nvidia Driver Version**: 440.3.01
**CUDA Version**: 10.2
**CUDNN Version**: 7.6.5
**Operating System + Version**: ubuntu 16.04
**Python Version (if applicable)**: 2.7
**TensorFlow Version (if applicable)**: X 
**PyTorch Version (if applicable)**: 1.5
**Baremetal or Container (if container which image + tag)**: X

## Steps To Reproduce

simple code below should reproduce this issue.

    
    import torch
    import torch.nn as nn
    import torch.nn.functional as F
    from torch2trt import torch2trt
    from torch2trt.torch2trt import *
    

    torch.manual_seed(0)

    class NonPoolingBlock(nn.Module):
        def __init__(self, in_channel, mid_channel, filter_size=3, padding=1):
            super(NonPoolingBlock, self).__init__()
            self.block1 = ConvBlock(in_channel, mid_channel)
            self.block2 = ConvBlock(mid_channel, in_channel)

        def forward(self, x):
            return x+ (self.block2(self.block1(x)))

    class ConvBlock(nn.Module):
        def __init__(self, in_channel, out_channel, filter_size=3, padding=1, stride=1):
            super(ConvBlock, self).__init__()
            self.conv = nn.Conv2d(in_channel, out_channel, filter_size, padding=padding, stride=stride)
            self.conv_bn = nn.BatchNorm2d(out_channel)

        def forward(self, x):
            x = F.relu(self.conv_bn(self.conv(x)))
            return x

    net = NonPoolingBlock(4,4).eval().cuda()
    data = torch.ones((1,4,1,1)).cuda()
    model_trt = torch2trt(net,[data])

    print(net(data))
    print(model_trt(data))
    print(torch.max(torch.abs(model_trt(data) - net(data))))
    
Output looks like below.
    
    tensor([[[[1.0527]],

            [[1.0000]],

            [[1.0000]],

            [[1.0000]]]], device='cuda:0', grad_fn=<AddBackward0>)
    tensor([[[[1.0527]],

            [[0.9966]],

            [[0.9625]],

            [[0.8973]]]], device='cuda:0')
    tensor(0.1027, device='cuda:0', grad_fn=<MaxBackward1>)
    
If you pad some operations after relu, like padding useless leaky_relu, relu applys well.

**Comment by ttyio** [comment]
Hello @HeekonKim , thanks for reporting and sorry for delay response.

If you export the onnx using

    torch.onnx.export(net, data, "output.onnx", opset_version=11)

then run the onnxruntime and trt using 

    polygraphy run --trt --onnxrt output.onnx

You can find the trt output matches onnxruntime output.

      [S]         PASSED | Difference is within tolerance (rtol=1e-05, atol=1e-05)
      [S] PASSED | Command: /home/vincenth/.local/bin/polygraphy run --trt --onnxrt output.onnx

so it seems `torch2trt` has some exporting bugs, could you create issue in https://github.com/NVIDIA-AI-IOT/torch2trt ? thanks!

**Comment by ttyio** [comment]
I will close since no response for more than 3 weeks, please reopen if you still have question, thanks!
